modules/nodes-containers-downward-api-container-resources-envars.adoc

```diff
@@ -9,17 +9,30 @@
 When creating pods, you can use the Downward API to inject information about
 computing resource requests and limits using environment variables.
 
+When creating the pod configuration, specify environment variables that
+correspond to the contents of the `resources` field in the `*spec.container*`
+field.
+
+[NOTE]
+====
+If the resource limits are not included in the container configuration, the
+downward API defaults to the node's CPU and memory allocatable values.
+====
+
 .Procedure
 
-To use environment variables:
+. Create a new pod spec that contains the resources you want to inject:
 
-. When creating a pod configuration, specify environment variables that
-correspond to the contents of the `resources` field in the `*spec.container*`
-field:
+.. Create a `pod.yaml` file similar to the following:
```

Comment:
I think this formatting can be tweaked a little bit. The example is on step 2, and step 1 only has an "a," not a "b." See RH supplementary style guide: https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures